_im-plugin/index-context.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: default
+title: Index context
+nav_order: 14
+redirect_from:
+  - /opensearch/index-context/
+---
+
+# Index context
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress the feature or if you want to leave feedback, join the discussion in the [OpenSearch forum](https://forum.opensearch.org/).    
+{: .warning}
+
+Index Context declares the use case for an index. Using the context information, OpenSearch applies a pre-determined set of settings and mappings to do the heavy-lifting to ensure. When given the correct context, context provides indexes with:
+
+- Optimized performance.
+- Settings tuned to your specific use case.
+- Accurate mappings and aliases based on [OpenSearch Integrations]({{site.url}}{{site.baseurl}}/integrations/).
+
+The settings and metadata configuration that are applied using component templates are automatically loaded when your cluster starts. These component templates start with `@abc_template@` or Application Based Configuration (ABC) templates. To prevent potential configuration issues, you can only use ABC templates the way it’s described on this page, through a `context` object declaration.
```

Comment:
```suggestion
The settings and metadata configuration that are applied using component templates are automatically loaded when your cluster starts. Component templates that start with `@abc_template@` or Application Based Configuration (ABC) templates can only be used through a `context` object declaration, in order to prevent potential configuration issues.
```